### DISCUSSION TITLE
Filament V3 announcement: Relationmanagers in the form using @teleport

### DISCUSSION THREAD

**Discussion by demianottema** [filamentphp/filament#11845] [General]
When Filament V3 was announced (#3654) i saw that moving relation managers to anywhere in the form like a formcomponnent was a thing. This could be done using the @teleport function. Since then i was super hyped, because now i use repeaters everywhere, that works, but i would love all the functionaities you get from a table. 

I was wondering if this is still not possible, or if it is much work. Or maybe i completly missed it, and it is possible, but then i think there is missing some docs. Because this would be a crucial functionality!

**Comment by SayHeyD** [comment]
Maybe this could help?

https://github.com/njxqlus/filament-relation-manager-component
